.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: c
+sudo: required
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
+script: bash -ex .travis-opam.sh
+env:
+  matrix:
+  - OCAML_VERSION=4.03
```

Comment:
I am not sure whether it is still the way to go in OPAM 2, but could you add `available: [ ocaml-version >= "4.03" ]` to the OPAM file, since we don't expect this version to build anyway?

Reply:
The dependency is implicitly there as the library depends on configuration, which is only installable for ocaml >= 4.03.0.
I'd leave it out for now.